lib/scenic/adapters/postgres.rb

```diff
@@ -17,6 +17,10 @@ def self.drop_view(name)
         execute "DROP VIEW #{name};"
       end
 
+      def self.extensions(base = ActiveRecord::Base)
```

Comment:
For databases that don't support extensions, `ActiveRecord:Base.connection` returns an empty array, so we would just end up with an empty array of extensions to check against.